src/app/admin/components/microschema-detail/mircoschema-detail.component.ts

```diff
@@ -17,9 +29,23 @@ import { MarkerData } from '../monaco-editor/monaco-editor.component';
 export class MicroschemaDetailComponent implements OnInit, OnDestroy {
     // TODO Disable save button when editor is pristine
     // TODO Show message on save when schema has not changed
+    projects: Project[];
+
+    filterTerm: string;
+
+    filterInput = new FormControl('');
```

Comment:
Check if filter is needed. If not, remove it.

Reply:
Or better: add the filter to the admin list

Reply:
Filter for Allocations Tab was added to:
- mircoschema-detail.component.ts;
- schema-detail.component.ts.